.travis.yml

```diff
@@ -50,8 +42,8 @@ install:
   # install matplotlib
   - pip install matplotlib
   # install warp-ctc
-  - git clone https://github.com/SeanNaren/warp-ctc.git 
-  - cd warp-ctc && git checkout 9e5b238f8d9337b0c39b3fd01bbaff98ba523aa5 && mkdir build && cd build && cmake .. && make -j4 && cd ..
+  - git clone https://github.com/SeanNaren/warp-ctc.git
```

Comment:
As I noted in `Makefile`, use follows-upstream branch of https://github.com/jnishi/warp-ctc of supports because it supports pytorch 0.4.1 and #307.